src/ZODB/tests/testFileStorage.py

```diff
@@ -456,6 +456,10 @@ def testRestoreAcrossPack(self):
         # Skip this check as it calls restore directly.
         pass
 
+    def testRestoreWithMultipleUndoRedo(self):
+        # This case is only supported with restore, not with store "emulation"
+        pass
```

Comment:
```suggestion
    @unittest.skip('only supported with restore, not with store "emulation"')
    def testRestoreWithMultipleUndoRedo(self):
        pass
```

This makes this test visible as skipped as it is not running at all.

Reply:
That crossed my mind as well. I just followed the same pattern as above:
 
https://github.com/zopefoundation/ZODB/blob/44d36b0599a2fcb709b5fd90b820f9c283a8fe53/src/ZODB/tests/testFileStorage.py#L455-L457

there are other places with some base test class having lots of tests and the test for specific storages override test method to do nothing: 

https://github.com/zopefoundation/ZODB/blob/6f8878124fa8bf1cc1932a8f67da8d15b653897c/src/ZODB/tests/testDemoStorage.py#L133-L138

https://github.com/zopefoundation/ZODB/blob/6f8878124fa8bf1cc1932a8f67da8d15b653897c/src/ZODB/tests/testMappingStorage.py#L46-L54

but it seems these tests were written before unittest had support for skip.

I can make another pull request to change all these to use skip, unless someone here thinks it's bad ?

Reply:
I'm resolving this without applying the change, because this might break -D option of testrunner https://github.com/zopefoundation/ZODB/pull/395#issuecomment-1921379301